paddle/trainer/tests/test_Trainer.cpp

```diff
@@ -90,7 +94,7 @@ TEST(checkGradient, multi) {
 TEST(checkGradient, hsigmoid) { checkGradientTest(configFile2, false, false); }
 
 TEST(checkGradient, chunk) {
-  EXPECT_EQ(0, system("python2 trainer/tests/gen_proto_data.py"));
+  EXPECT_EQ(0, system("python trainer/tests/gen_proto_data.py"));
```

Comment:
MAC OS X do not have python2. I will add some condition branch to avoid potential hazard.